libs/net/dal/Extensions/DbContextExtensions.cs

```diff
@@ -122,6 +129,10 @@ public static TNOContext OnUpdate<T>(this TNOContext context, T entity, ClaimsPr
             cache.Value = Guid.NewGuid().ToString();
             context.Update(cache);
         }
+        else
+        {
+            context.Add(new Cache(key, Guid.NewGuid()));
```

Comment:
Adding it to cache is helpful to handle situations where we haven't created an entry for the cache table.